### PR TITLE
#423 2: Electric Boogaloo

### PR DESCRIPTION
**Pull request by ErwinSnowden**
## About The Pull Request
does what #423 does, this is a necessary change to prevent powergaming.

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl:
balance: Wearing heavy helmets without support is a very bad idea!
/:cl:

### CODE REVIEW

**Comment by BadAtThisGame302** [comment]
🙏🏻

**Comment by Al-1ce** [comment]
Inshallah this will be merged.

File: code/modules/mob/mob_movement.dm [hunk 1]
```diff
@@ -83,6 +83,15 @@
 	if(SEND_SIGNAL(mob, COMSIG_MOB_CLIENT_PRE_MOVE, args) & COMSIG_MOB_CLIENT_BLOCK_PRE_MOVE)
 		return FALSE
 
+	if (istype(L.get_item_by_slot(SLOT_HEAD), /obj/item/clothing/head/helmet/f13/power_armor) || istype(L.get_item_by_slot(SLOT_HEAD), /obj/item/clothing/head/helmet/f13/heavy/salvaged_pa))
```
Comment (out-of-phaze) [inline code/modules/mob/mob_movement.dm:86]: hahahahahahaha you straight up put it in fucking `client/Move` instead of using signals hahahahahahahahahaha

Comment (Al-1ce) [reply code/modules/mob/mob_movement.dm:86]: this is not helpful

**Pull request closed without merging**